lbz/dev/test.py

```diff
@@ -83,7 +83,7 @@ def _process(
                     query_params[key] = [str(value)]
                 else:
                     query_params[key] = [str(elem) for elem in value]
-        return self.resource(
+        res: Response = self.resource(
```

Comment:
I would use a full name for this variable as it looks like you create a resource at the first glance:
```suggestion
        response: Response = self.resource(
```